### DISCUSSION TITLE
Handlebars inArray not working as expected

### DISCUSSION THREAD

**Discussion by R2bEEaton** [Budibase/budibase#6618] [Help] [answered]
Hello!

I have a card that I would like to hide based on whether a certain value exists in another table. The following is my handlebar expression to see if it does:

`{{ inArray ( pluck Source.Rows "TripNbr" ) Card.TripNbr }}`

This, however, always returns false even when I know the TripNbr exists in the other table.

For example:
```
{{ pluck Source.Rows "TripNbr" }} -> 5948,6223
{{ Card.TripNbr }} -> 5948
{{ inArray ( pluck Source.Rows "TripNbr" ) Card.TripNbr }} -> false
{{ inArray ( pluck Source.Rows "TripNbr" ) "5948" }} -> true
```

Using JavaScript, I checked the types of these elements:
```
typeof $("Source.Rows.0.TripNbr") -> string
typeof $("Card.TripNbr") -> number
```

When I thought that was the problem, I then used the following Handlebars expression to no avail:
`{{ inArray ( map ( pluck Source.Rows "TripNbr" ) toInt ) Card.TripNbr }}`

I want to be clear that I know how to do this with JavaScript, but using Handlebars would be preferable for maintaining purposes. I've been scratching my head for a while on this one and would really appreciate any insight. Thanks!

**Comment by melohagan** [comment, marked as answer]
HI @R2bEEaton 

Instead of trying to cast the array to an int, have you tried casting `Card.TripNbr` to a string?

This worked for me:
`{{ inArray ( pluck Source.Rows "TripNbr" ) ( stringify Card.TripNbr ) }}`

**Reply by R2bEEaton** [reply]
That worked perfectly, thank you very much! I had seen `stringify` but assumed it was only for JSON objects.
